Business Rules/Backup Critical Table Data/backupCriticalTableData.js

```diff
@@ -0,0 +1,18 @@
+// Script to back up critical table data to external storage
+(function executeRule(current, previous /*null when async*/) {
+    var recordData = {
+        sys_id: current.sys_id.toString(),
+        number: current.number.toString(),
+        short_description: current.short_description.toString()
```

Comment:
In future, use getter methods (`getValue('field')` and `getUniqueValue()`) to avoid having to use this method